app/views/people/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @purchased_gifts.each do |purchased_gift| %>
+  <div data-role="name"><%= purchased_gift.name %></div>
+  <div data-role="url"><%= purchased_gift.url %></div>
+  <div data-role="amount"><%= number_to_currency(purchased_gift.amount) %></div>
+<% end %>
+
+<%= link_to("Add purchased_gift",
```

Comment:
This has a really weird break. What do you think about letting this just run long?